worlds/tunic/__init__.py

```diff
@@ -74,6 +95,56 @@ def generate_early(self) -> None:
                 self.options.entrance_rando.value = passthrough["entrance_rando"]
                 self.options.shuffle_ladders.value = passthrough["shuffle_ladders"]
 
+    @classmethod
+    def stage_generate_early(cls, multiworld: MultiWorld) -> None:
+        tunic_worlds: Tuple[TunicWorld] = multiworld.get_game_worlds("TUNIC")
+        for tunic in tunic_worlds:
+            if isinstance(tunic.options.entrance_rando.value, str):
+                group = tunic.options.entrance_rando.value
+                # if this is the first world in the group, set the rules equal to its rules
+                if group not in cls.seed_groups:
```

Comment:
Again, here I would write `continue` at the end of the clause and then remove the `else:` to lose an indentation level